test/parallel/xyk-rpc.errors.test.ts

```diff
@@ -117,4 +118,25 @@ test.each([
 
 
 
+test.each([
+	[new BN('100000000')],
+	[new BN('100000000000000')],
+	[new BN('10000000000000000000000')] //<- Fails with:
```

Comment:
@mateuszaaa ,
following your comment in trello, I thought that there is hope, but still failing :(
I run out of ideas and I think @iStrike7  too.
any clue ?

Reply:
solved by Mat.